src/actions/order-actions.js

```diff
@@ -846,7 +846,9 @@ export const saveOrderExtraQuestionsSubQuestionsRule =
     dispatch(startLoading());
 
     const params = {
-      access_token: accessToken
+      access_token: accessToken,
+      expand: "sub_question",
+      fields: "sub_question.name"
```

Comment:
its still retrieving not needed data
please do
 **relations: "sub_question.none"**